specification/Metadata/README.md

```diff
@@ -0,0 +1,446 @@
+| INT16   | Signed integer in the range `[-32768, 32767]`                             |
+| UINT16  | Unsigned integer in the range `[0, 65535]`                                |
+| INT32   | Signed integer in the range `[-2147483648, 2147483647]`                   |
+| UINT32  | Unsigned integer in the range `[0, 4294967295]`                           |
+| INT64   | Signed integer in the range `[-9223372036854775808, 9223372036854775807]` |
+| UINT64  | Unsigned integer in the range `[0, 18446744073709551615]`                 |
+| FLOAT32 | A number that can be represented as a 32-bit IEEE floating point number   |
+| FLOAT64 | A number that can be represented as a 64-bit IEEE floating point number   |
+| BOOLEAN | True or false                                                             |
+| STRING  | A sequence of characters                                                  |
+| ENUM    | An enumerated type                                                        |
+
+Floating-point properties (`FLOAT32` and `FLOAT64`) must not include values `NaN`, `+Infinity`, or `-Infinity`.
+
+[Enum properties](#enums) are denoted by `ENUM`. An enum property must additionally provide the ID of the specific enum it uses, referred to as its enum type (`enumType`).
+
+> **Implementation Note:** Developers of authoring tools should be aware that many JSON implementations support only numeric values that can be represented as IEEE-754 double precision floating point numbers. Floating point numbers should be representable as double precision IEEE-754 floats when encoded in JSON. When those numbers represent property values (such as `noData`, `min`, or `max`) having lower precision (e.g. single-precision float or 8-bit or 16-bit integer), the values should be rounded to the same precision in JSON to avoid any potential boundary mismatches. Numeric property values encoded in binary storage are unaffected by these limitations of JSON implementations.
+
+#### Normalized Values
+
+Normalized properties (`normalized`) provide a compact alternative to larger floating-point types. Normalized values are stored as integers, but when accessed are transformed to floating-point form according to the following rules:
+
+* Unsigned integer values (`UINT8`, `UINT16`, `UINT32`, `UINT64`) must be rescaled to the range `[0.0, 1.0]` (inclusive)
+* Signed integer values (`INT8`, `INT16`, `INT32`, `INT64`) must be rescaled to the range `[-1.0, 1.0]` (inclusive)
+
+> **Implementation Note:** Depending on the implementation and the chosen integer type, there may be some loss of precision in values after denormalization. For example, if the implementation uses 32-bit floating point variables to represent the value of a normalized 32-bit integer, there are only 23 bits in the mantissa of the float, and lower bits will be truncated by denormalization. Client implementations should use higher precision floats when appropriate for correctly representing the result.
```

Comment:
Not 100% if it's "unnormalize" or "denormalize", but we've been using "unnormalize" in our impelementation. 

```suggestion
> **Implementation Note:** Depending on the implementation and the chosen integer type, there may be some loss of precision in values after unnormalizing. For example, if the implementation uses 32-bit floating point variables to represent the value of a normalized 32-bit integer, there are only 23 bits in the mantissa of the float, and lower bits will be truncated after unnormalizing. Client implementations should use higher precision floats when appropriate for correctly representing the result.
```

Reply:
I think I disagree with this change – the two words have different meanings and this is the correct one for the context:

- unnormalized: value has not been normalized.
- denormalized: value has been decoded from a normalized form.

Wikipedia makes similar distinctions between [denormalized](https://en.wikipedia.org/wiki/Denormalization) and [unnormalized](https://en.wikipedia.org/wiki/Unnormalized_form) forms, referring to relational tables and not GPU attributes though.

Reply:
okay, leave it as-is.